tests/foreman/cli/test_leapp_client.py

```diff
@@ -126,18 +126,20 @@ def leapp_repos(
         release_version = RHEL_REPOS[rh_repo_key]['releasever']
         if release_version in str(source) or release_version in target:
             prod = 'rhel' if 'rhel7' in rh_repo_key else rh_repo_key.split('_')[0]
+            repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+                basearch=default_architecture.name,
+                org_id=module_sca_manifest_org.id,
+                product=PRDS[prod],
+                repo=RHEL_REPOS[rh_repo_key]['name'],
+                reposet=RHEL_REPOS[rh_repo_key]['reposet'],
+                releasever=release_version,
+            )
             if module_stash[synced_repos].get(rh_repo_key, None):
+                rh_repo = module_target_sat.api.Repository(id=repo_id).read()
+                all_repos.append(rh_repo)
```

Comment:
Looks, these 2 lines can move out of if...else... block
We can keep logger inside if block (for better understanding) and sync repo inside else block.

Reply:
I would like to keep it as it is.